KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,60 @@
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help grow the community by bringing new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
+5. Maintain the Kubeflow Website, documentation, demonstration, and educational material for the Kubeflow community.
+
+### Committee Structure
+
+- **Appointment of Members**: Members of the Kubeflow Outreach Committee are appointed at the discretion of the Kubeflow Steering Committee (KSC), with the removal of any members for violating the CNCF Code of Conduct also at the KSC's discretion through a majority vote.
+- **Role of KOC Co-Leads**: The Outreach Committee will be co-led by established Kubeflow community members, with the co-leads being responsible for guiding the committee’s work. New members of the KOC may be newcomers to the community, and their primary role will be focused on helping drive growth for Kubeflow.
+- **Committee Expansion**: KOC may appoint new members as needed, requiring only the approval of one KSC member for the appointment.
+
+## Committee members
+
+KOC is composed of 3 (three) members. Seats on the Outreach Committee are held by an individual, not by their employer.
+
+### Limitations on Company Representation
+No more than one seat may be held by employees of the same organization (or conglomerate, in the case of companies owning each other). 
+
+
+### Committee Meetings
+
+The KOC will meet regularly (at least monthly, or as needed), and these meetings will be open to the public. Specific sensitive matters, such as code of conduct violations or confidential discussions, may be handled separately.
+
+Meeting notes will be publicly available to the community, except when privacy is required for specific discussions.
+
+### Decision Process
+
+The KOC aims to reach consensus on all decisions. If a decision requires a vote, the decision will pass with a simple majority of the members present. 
+
+Decisions are made in meetings when a quorum of the members are present and may pass with at least half the members (rounded up) of the committee supporting it.
```

Comment:
Do leads participate in decision process ?

Reply:
I think they should be heard by KOC but KOC is the decision maker.

Reply:
We should add it to the charter.